modules/uhttp/http.go

```diff
@@ -114,20 +115,22 @@ func newModule(
 
 	handlers := addHealth(getHandlers(mi.Host))
 
+	log := ulog.Logger(context.Background()).With("moduleName", mi.Name)
+
 	// TODO (madhu): Add other middleware - logging, metrics.
 	module := &Module{
 		ModuleBase: *modules.NewModuleBase(mi.Name, mi.Host, []string{}),
 		handlers:   handlers,
-		fcb:        defaultFilterChainBuilder(mi.Host),
+		fcb:        defaultFilterChainBuilder(log, mi.Host.AuthClient()),
```

Comment:
`SetLogger` updates the global logger, we don't want to set it to be uhttp specific.